### ISSUE TITLE
All specifications show on each page for DSS

### ISSUE THREAD

**Issue by robhrt7** [sourcejs/sourcejs-contrib-dss#3]
Moved issue from other project https://github.com/sourcejs/Source/issues/213

@custa1200:

> Hi i am using the latest nightly build and am finding as I am buiding up my specs that they are all appearing on each page.

**Comment by robhrt7** [comment]
From chat, file structure:

![image](https://photos-5.dropbox.com/t/2/AADTNBsw67tj-ZYGBZdT1NWersQWLYVHU47Ij2eAFl6usQ/12/33734426/png/32x32/3/1459263600/0/2/Screenshot%202016-03-29%2020.18.39.png/EOfMyhkY1OMJIAIoAg/wCUscOjO-I8fWRi_ZAU6lag809HL1DKvSH_xRwVOzZk?size_mode=3&size=2048x1536)

**Comment by robhrt7** [comment]
Tried to change mask, but did not helped

```
plugins: {
        dss: {
           targetCssMask: 'css/component.styl'
        }
    }
```

**Comment by robhrt7** [comment]
Fixed on 0.1.1 https://github.com/sourcejs/sourcejs-contrib-dss/commit/2990505dcdfd20700170b7f43f52b8ea76037b46
